pkg/kubelet/status/status_manager_test.go

```diff
@@ -552,7 +552,7 @@ func TestStaticPod(t *testing.T) {
 
 	t.Logf("Should not sync pod in syncBatch because there is no corresponding mirror pod for the static pod.")
 	m.syncBatch(true)
-	assert.Equal(t, len(m.kubeClient.(*fake.Clientset).Actions()), 0, "Expected no updates after syncBatch, got %+v", m.kubeClient.(*fake.Clientset).Actions())
+	assert.Empty(t, m.kubeClient.(*fake.Clientset).Actions(), "Expected no updates after syncBatch, got %+v", m.kubeClient.(*fake.Clientset).Actions())
```

Comment:
Is dumping `m.kubeClient.(*fake.Clientset).Actions()` redundant here?

In other words, does `assert.Empty` print the thing that is not empty?

Reply:
Yes, that's what I was also thinking. The message is fine, just not that it repeats the content of the item. That is no longer necessary.